odl/space/base_tensors.py

```diff
@@ -968,8 +968,9 @@ def show(self, title=None, method='', indices=None, force_show=False,
 
         # Default to showing x-y slice "in the middle"
         if indices is None and self.ndim >= 3:
-            indices = (slice(None),) * 2
-            indices += tuple(n // 2 for n in self.space.shape[2:])
+            indices = tuple(
+                [slice(None)] * 2 + [n // 2 for n in self.space.shape[2:]]
```

Comment:
No need to create a list, the generator is enough

Reply:
I can't do `* 2` or `+` with a generator, that would require chaining and stuff. This is the simplest implementation, I think.

Reply:
Use parentheses